lib/runtime/invalid_transaction_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package runtime
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/transaction"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInvalidTransactionErrors(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		name        string
+		test        []byte
```

Comment:
nit a more explicit name like `encoded` / `encodedData` would be nicer than `test`